ui/app/components/app/asset-list/asset-list.js

```diff
@@ -57,6 +57,7 @@ const AssetList = ({ onClickAsset }) => {
         onClick={() => onClickAsset(nativeCurrency)}
         data-testid="wallet-balance"
         primary={primaryCurrencyDisplay}
+        tokenSymbol="ETH"
```

Comment:
I'm not sure this is useful anymore, once it's no longer used to determine whether an item is a token.